elastic/kibana
# [SecuritySolution] Check visualize.save on added to library

## Summary

Previously we use `canUseEditor` to check if we should disable the `Added to Library` action. It's not ideal as it checks `core.application.capabilities.visualize?.show` under the hood.
We should use `application.capabilities.visualize?.save` to make sure `Added to Library` is disabled when they have no rights to save a visualization.

Steps to verify:

1. Create a role with read visualization privilege, and assign it to a user:
<img width="2556" alt="Screenshot 2024-10-02 at 13 20 44" src="https://github.com/user-attachments/assets/1ab9ddcf-96fd-4fd1-bdad-7382573350fb">

2. Login with the user and check `Add to Library` should be disabled:
<img width="2556" alt="Screenshot 2024-10-02 at 13 20 11" src="https://github.com/user-attachments/assets/9681b121-77e6-47c1-9a99-57d53f5d0b07">


### Checklist

Delete any items that are not applicable to this PR.
